documentation/general/decouple-vs-and-net-sdk.md

```diff
@@ -136,8 +136,11 @@ To mitigate this we will be fixing the `build-server shutdown` command to be rel
 Based on the value of the `RoslynCompilerType` property, the SDK sets property `RoslynTasksAssembly` to a full path to a [Roslyn build task DLL][roslyn-build-task],
 and the SDK targets use `$(RoslynTasksAssembly)` to load the build task.
 
-The SDK also sets `RoslynTargetsPath` to the directory path of the roslyn tasks assembly. This property is used by some targets
-but it should be avoided if possible because the tasks assembly name can change as well, not just the directory path.
+The SDK also sets the following properties:
+- `RoslynTargetsPath` to the directory path of the roslyn tasks assembly. This property is used by some targets
+  but it should be avoided if possible because the tasks assembly name can change as well, not just the directory path.
+- `RoslynAssembliesPath` to the directory path of other roslyn assemblies (like `Microsoft.CodeAnalysis.dll`).
```

Comment:
@ericstj this value can point to assemblies that target `net472` or .NET core. Will the API compat tool be okay with both of these?

Reply:
I would expect it to point to the correct framework for the MSBuild host process.  So if MSBuild is running as netfx, point to netfx assemblies.  If MSBuild is running as core, point to core assemblies.

If you think there is a scenario for cross-framework execution (IE: if MSBuild might allow a core-taskhost in the future) then we should also have specific properties that always point at core and netfx.

Reply:
Interesting. Why did you pick this? My intuition is that `RoslynAssembliesPath` would point to the assemblies that would be used by the build task.

Reply:
I can go ahead and add a property for core-only roslyn assemblies even now.